CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Welcome!
+
+These are the ways you can currently contribute to the OSI Samples:
+
+- Give us [Feedback](https://feedback.osisoft.com/forums/922279-osisoft-github) at https://Feedback.osisoft.com/forums/922279-osisoft-github.  Please select the appropriate category when posting feedback.  This feedback can include new features, issues, requests for new samples, etc...
```

Comment:
Not sure I would put the same link back to back like this. I think you wanted to show the whole URL, which is fine, I'd just say "Give us feedback (<- not a URL) at https://feedback...."